cmake/FindSenseGlove.cmake

```diff
@@ -0,0 +1,34 @@
+# Copyright 2020 Istituto Italiano di Tecnologia (IIT)
+# @author Kourosh Darvish <kourosh.darvish@iit.it>
+
+# Finds the Sense Glove SDK
+#
+# This will define the following variables::
+#
+#   SenseGlove_FOUND         - True if the system has the Sense Glove SDK
+#   SenseGlove_INCLUDE_DIR  - The include directory of the SDK
+#   SenseGlove_LIBRARIES     - The name of the libraries to link against.
+###############################
+
+# Check Directory of the SenseGlove_DIR
+if(NOT DEFINED ENV{SenseGlove_DIR})
+  message( FATAL_ERROR "Environment variable {SenseGlove_DIR} is not defined: $ENV{SenseGlove_DIR}" )
+else()
+  message("Environment variable {SenseGlove_DIR}: $ENV{SenseGlove_DIR}" )
+endif()
+
+if(NOT DEFINED ENV{SenseGlove_INCLUDE_DIRS})
```

Comment:
Could this be inferred from `$ENV{SenseGlove_DIR}`?

Reply:
Here, `SenseGlove_DIR` is the directory that we have the libraries; this is passed to `PATH` as well.
So it is not the root of the git repo related to senseGlove. If I want to infer it, I sould add sth like `../../<path to header files>`.
Then if I want to add `SenseGlove_DIR` as the root of the repo, both for the `PATH` environment variable and inside the cmake file, I should add the folder. 
What is your suggestion?

cc @traversaro  @S-Dafarra

Reply:
An option might be also, to clone the senseGlove repo with superbuild, if the option is enabled.

Reply:
I think I am missing something. Is the relative path between ``SenseGlove_DIR`` and ``SenseGlove_INCLUDE_DIRS`` always fixed?

Also, if they have a common parent, it may also make sense to define that parent as ``SenseGlove_DIR``. For example, assume that the structure is as follows
```
- Senseglove_SDK_DIR
              |
              |-- bin
                    |
                    |-- someLib.dll
              |
              |-- include
                      |
                      |-- Senseglove
                                    |
                                    |-- SenseGlove.h
```
you could ask the user to set ``SenseGlove_DIR`` equal to the path of the Senseglove_SDK_DIR, then you hardcode the path to ``someLib.dll`` and ``SenseGlove.h`` starting from the ``SenseGlove_DIR``.

Reply:
I have updated the cmake file. Let me know if it looks fine.

Reply:
@S-Dafarra that is exactly what I meant, looks much better now!

Reply:
Can I resolve this conversation, I have updated the cmake files.